### ISSUE TITLE
App shows some connections even though they are canceled

### ISSUE THREAD

**Issue by WorldCodeCentral** [grote/Transportr#354]
Hi, first of all, thanks for this app, it's very useful to me. However today I checked for a connection from Mönchengladbach to Aachen and it showed me the usual connections. However when I clicked on them it says "Stellwerksstörung in Rheydt Hbf. Beeinrächtigungen zw. Herzogenrath und Mönchengladbach Hbf. Störungsdauer bis mind. 15:00 Uhr. Nutzen sie zw. ..." blabla and something about alternatives by bus that replace the trains.

I don't think these replacements will be as fast or easy as the original train. But even if they're as fast I'd like at least a warning symbol or something to show that the found train doesn't go there as usual.

Thanks and best regards

**Comment by grote** [comment]
Do you have maybe screenshots to illustrate this? Transportr should show red text below the connection if there's a problem.

The message itself is coming form the transport company's API and Transportr doesn't have any influence on this.

**Comment by WorldCodeCentral** [comment]
Hello! Hmm I have a problem with my SD card so can't easily take screenshots right now. However that wouldn't help anyway, because my problem is that the connection shows up as every other connection does, although there are problems. Transportr does show the red text, but only once I clicked on a connection. Not on the overview of connections view. So this is something that can be fixed through the app, because the information from the transport company is already there. I just wish Transportr would show that there is additional information from the transport company at the overview screen where all the results are shown, because I thought nothing was wrong until I clicked on one of them. I was pretty lucky that I noticed it at all :-)
